.github/pull_request_template.md

```diff
@@ -1,5 +1,35 @@
-# Proposed changes
+# Description
 
 Fixes # (issue)
 
 Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+## How to validate
+
+Please provide step-by-step instructions to replicate your validation scenario. For bug fixes, detail how to reproduce both the bug and its fix, along with any observations.
+
+1. 
+1. 
+1. 
+
+## CLI Quality Reminders 🔧
+
+For both authors and reviewers:
+
+- Scrutinize for potential regressions
+- Ensure key automated tests are in place
+- Build the CLI and test using the validation steps
+- Assess Developer Experience impact (log messages, performances, etc)
+- If too broad, consider breaking into smaller PRs
```

Comment:
link here https://github.com/okteto/okteto/blob/master/docs/code-review.md#pr-scope I think could be a good idea

Reply:
good shout, I will move the PR to draft and review that one too 👍